packages/peregrine/lib/talons/CreateAccount/createAccount.gql.js

```diff
@@ -0,0 +1,126 @@
+import { gql } from '@apollo/client';
+
+export const CREATE_ACCOUNT = gql`
+    mutation CreateAccount(
+        $email: String!
+        $firstname: String!
+        $lastname: String!
+        $password: String!
+        $is_subscribed: Boolean!
+    ) {
+        createCustomer(
+            input: {
+                email: $email
+                firstname: $firstname
+                lastname: $lastname
+                password: $password
+                is_subscribed: $is_subscribed
+            }
+        ) {
+            # The createCustomer mutation returns a non-nullable CustomerOutput type
+            # which requires that at least one of the sub fields be returned.
+            customer {
+                id
+            }
+        }
+    }
+`;
+
+export const GET_CUSTOMER = gql`
+    query GetCustomerAfterCreate {
+        customer {
+            id
+            email
+            firstname
+            lastname
+            is_subscribed
+        }
+    }
+`;
+
+export const SIGN_IN = gql`
+    mutation signIn($email: String!, $password: String!) {
+        generateCustomerToken(email: $email, password: $password) {
+            token
+        }
+    }
+`;
+
+export const CREATE_CART = gql`
+    mutation CreateCartAfterAccountCreation {
+        cartId: createEmptyCart
+    }
+`;
+
+export const GET_CART_DETAILS = gql`
+    query GetCartDetailsAfterAccountCreation($cartId: String!) {
+        cart(cart_id: $cartId) {
+            id
+            items {
+                id
+                prices {
+                    price {
+                        value
+                    }
+                }
+                product {
+                    id
+                    name
+                    sku
+                    small_image {
+                        url
+                        label
+                    }
+                    price {
+                        regularPrice {
+                            amount {
+                                value
+                            }
+                        }
+                    }
+                }
+                quantity
+                ... on ConfigurableCartItem {
+                    configurable_options {
+                        id
+                        option_label
+                        value_id
+                        value_label
+                    }
+                }
+            }
+            prices {
+                grand_total {
+                    value
+                    currency
+                }
+            }
+        }
+    }
+`;
+
+export const MERGE_CARTS = gql`
```

Comment:
This mutation previously injected multiple component fragments into the response which I removed, under the assumption I would re-add when we tackle migrating component colocated queries to peregrine. To my surprise, I didn't spot any stale cache issues after merge with just this response shape; I suspect Apollo v3 now correctly re-runs those queries for us.

We can still re-add those fragments once components come over, but just wanted to point out this seems to be working as-is.

Reply:
Add a TODO so we don't forget to re-add

Reply:
This PR got cut off at just migration of operations in `venia-ui/lib/queries`. This operation used four very large fragments that we'll gradually add back as migrated. Good reminder though to add this detail to those followup tickets.